lib/fog/compute/xen_server.rb

```diff
@@ -3,9 +3,9 @@
 module Fog
   module Compute
     class XenServer < Fog::Service
-      autoload :Models, 'fog/compute/xen_server/models'
-      autoload :Real, 'fog/compute/xen_server/real'
-      autoload :Mock, 'fog/compute/xen_server/mock'
+      autoload :Models, File.expand_path('../xen_server/models', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.